src/Bindings/Plugin.h

```diff
@@ -86,6 +86,7 @@ class cPlugin
 	virtual bool OnPlayerPlacingBlock       (cPlayer & a_Player, const sSetBlock & a_BlockChange) = 0;
 	virtual bool OnPlayerRightClick         (cPlayer & a_Player, int a_BlockX, int a_BlockY, int a_BlockZ, char a_BlockFace, int a_CursorX, int a_CursorY, int a_CursorZ) = 0;
 	virtual bool OnPlayerRightClickingEntity(cPlayer & a_Player, cEntity & a_Entity) = 0;
+	virtual bool OnPlayerClickingAtEntity   (cPlayer & a_Player, cEntity & a_Entity, const Vector3f & a_TargetPos) = 0;
```

Comment:
Please alpha-sort.
Also, is this a right-click or left-click? Might want to rename.

Reply:
No, I re-verified my code. It could be whatever according to https://wiki.vg/Protocol#Use_Entity
By default I assume it's a right click for the 1.8 protocol
And it parse which hand is it in 1.9 protocol